crop.go

```diff
@@ -41,16 +41,17 @@ import (
 	"os"
 	"time"
 
-	"code.google.com/p/draw2d/draw2d"
 	"github.com/lazywei/go-opencv/opencv"
+	"github.com/llgcode/draw2d/draw2dimg"
+	"github.com/llgcode/draw2d/draw2dkit"
 	"github.com/nfnt/resize"
 )
 
 var skinColor = [3]float64{0.78, 0.57, 0.44}
 
 const (
-	detailWeight = 0.2
-	faceDetectionHaarCascade = "/usr/share/opencv/haarcascades/haarcascade_frontalface_alt.xml"
+	detailWeight             = 0.2
+	faceDetectionHaarCascade = "./files/haarcascade_frontalface_alt.xml"
```

Comment:
You are right. How do you think about breaking the API and do something like this:

```go
NewCropWithHaarCascade(filename string) (Crop, error) {
....
}

type Crop interface {
   SmartCrop(img image.Image, width, height int) (Crop, error)
}

That way the user must provide correct arguments at bootstrap and we can validate that everything works at bootstrap.

Currently if that file is not found, my whole application stops with os.Exit(1) which is hardcoded :)